geode-core/src/main/java/org/apache/geode/cache/client/internal/ClientSideHandshakeImpl.java

```diff
@@ -233,7 +230,7 @@ public ServerQueueStatus handshakeWithServer(Connection conn, ServerLocation loc
 
       member = readServerMember(dis);
 
-      serverQStatus = new ServerQueueStatus(endpointType, queueSize, member);
+      ServerQueueStatus serverQStatus = new ServerQueueStatus(endpointType, queueSize, member);
```

Comment:
make `final`

Reply:
Could be said for so many places. What made you pick this one place I missed. ;)

Reply:
I have to make at least one `final` comment on every PR. Also this one just jumped out at me.